lib/travis/api/v3/models/log_part.rb

```diff
@@ -1,5 +1,6 @@
 module Travis::API::V3
   class Models::LogPart < Model
+    establish_connection(Travis.config.logs_database)
```

Comment:
IIRC, in general, it is. The Log and LogPart models need to be told to talk to the logs database. I'm not sure about the implementation in v3, but this looks similar to what we're doing in other apps.